feedback/templates/feedback/feedback_form.html

```diff
@@ -0,0 +1,42 @@
+{% extends "minimal.html" %}
+{% load i18n %}
+{% block title %}{{ block.super }} &mdash; {% trans "Give Us Feedback" %}{% endblock title %}
+{% block main %}
+            <div class="feedback-body">
```

Comment:
Why so much indentation hee?

Reply:
It matches the indentation in the parent thing. Is that a bad thing?

Reply:
Interested. I have seen this reasoning before and I have used it myself. But I guess the child should not be aware of the parents' indentation?